src/extensions/agent_modules/cloud_communication/eg_mqtt_broker_client/src/adu_enrollment_management.c

```diff
@@ -631,7 +526,7 @@ bool EnrollmentStatusRequestOperation_DoWork(ADUC_Retriable_Operation_Context* c
     return opSucceeded;
 }
 
-bool EnrollmentRequestOperation_CancelOperation(struct ADUC_Retriable_Operation_Context_t* context)
+bool EnrollmentRequestOperation_CancelOperation(ADUC_Retriable_Operation_Context* context)
```

Comment:
should these structs be const?  Or are they inout?

Reply:
This is potentially inout because of runtime data fields like retryParamCount, state (operation state), lastExecutionTime, lastFailureTime, lastSuccessTime, lastErrorContext, etc.

I might factor these out so that most of it could be const struct and the runtime state would be in a separate inout struct, but not in this PR